doc/source/whatsnew/v1.0.0.rst

```diff
@@ -204,286 +204,288 @@ Other enhancements
 - Roundtripping DataFrames with nullable integer or string data types to parquet
   (:meth:`~DataFrame.to_parquet` / :func:`read_parquet`) using the `'pyarrow'` engine
   now preserve those data types with pyarrow >= 1.0.0 (:issue:`20612`).
+- :meth:`DataFrame.sort_values`, :meth:`DataFrame.sort_index`, :meth:`Series.sort_index`, and :meth:`Series.sort_index`
+   now support the ``key`` argument which allows for custom sorting orders (:issue:`3942`)
 - The ``partition_cols`` argument in :meth:`DataFrame.to_parquet` now accepts a string (:issue:`27117`)
 - :func:`to_parquet` now appropriately handles the ``schema`` argument for user defined schemas in the pyarrow engine. (:issue: `30270`)
 
 
 Build Changes
-^^^^^^^^^^^^^
+   ^^^^^^^^^^^^^
```

Comment:
Not sure what happened here but can you revert all of this indentation?

Reply:
fixed, must have happened by mistake in some formatting change.